modules/BTBidAdapter.md

```diff
@@ -0,0 +1,40 @@
+# Overview
+
+**Module Name**: BT Bidder Adapter
+**Module Type**: Bidder Adapter  
+**Maintainer**: engsupport@blockthrough.com
+
+# Description
+
+The BT Bidder Adapter provides an interface to the BT Service. The BT Bidder Adapter sends one request to the BT Service per ad unit, but the BT Service will then send one request to each configured exchange. This is functionally similar to Prebid Server. One client side request, bids from many exchanges.
+
+The BT service accepts industry standard OpenRTB 2.5 as its payload. The BT Bidder Adapter converts Prebid.js parameters into an OpenRTB 2.5 request.
```

Comment:
so, this is just copy-pasted from Relay docs, we might be more creative, unless this is approved by Product